app/javascript/components/admin/AdminTable.js

```diff
@@ -521,18 +523,29 @@ class AdminTable extends React.Component {
 
     const handleSuccess = response => {
       if (response && response.data && response.data.user_rows) {
+        const keysToSkip = ['active_state', 'lock_reason', 'auto_lock_reason'];
         // NOTE: react-csv has a bug where false values don't show up in the downloaded CSV
         // This has been addressed and recently merged: https://github.com/react-csv/react-csv/pull/193
         // Once this gets released and we update our version this code won't be necessary.
-        const csvData = response.data.user_rows.map(userData => {
-          return {
-            ...userData,
-            notes: removeFormulaStart(userData.notes).toString(),
-            is_api_enabled: userData.is_api_enabled.toString(),
-            is_2fa_enabled: userData.is_2fa_enabled.toString(),
-            is_locked: userData.is_locked.toString(),
-          };
+        const keysWithBoolValues = ['is_api_enabled', 'is_2fa_enabled', 'is_locked'];
```

Comment:
In the JIRA ticket:
```
Ensure the renamed "Status" --> "System Access" header is updated correctly in the export.
```

I'm seeing the `System Access` header say `is_locked`.  Does this need to be updated?

Reply:
This is what I see in the current version of the app (i.e., we don't have "human-readable" headers). I can definitely ask @hackrm if these should be changed! 

<img width="795" alt="Screen Shot 2022-01-07 at 5 48 21 PM" src="https://user-images.githubusercontent.com/2328582/148617131-d6000855-156c-4511-8c45-8d93e7e09087.png">

Reply:
This is the current Admin Export. It looks like we don't currently have human-readable headings, so I didn't make a text change to is_locked. I can follow-up with @hackrm to see what she expects.

<img width="795" alt="Screen Shot 2022-01-07 at 5 48 21 PM" src="https://user-images.githubusercontent.com/2328582/148617430-1f6606ec-a4a3-487a-91a6-bc44d53a878b.png">

Reply:
hmm, since the header renaming for the rest of the columns will be handled in this [ticket](https://tracker.codev.mitre.org/browse/SARAALERT-488), it may be best to hold off on updating just this header so all the headers can be renamed at once. So it's probably best to not include this rename in this PR. I will strikeout that part of the DoD. Thanks for the input!